core/src/labels/label.h

```diff
@@ -186,6 +191,8 @@ class Label {
 
     int m_anchorIndex;
 
+    uint32_t m_selectionColor;
```

Comment:
I think we don't need that member at all when `getSelectionColor() { return m_vertexAttribs.selectionColor; }` is defined for Sprite/TextLabels :)